chainer/train_imagenet.py

```diff
@@ -64,45 +91,38 @@ def train_loop():
     label = np.ndarray((args.batchsize), dtype=np.int32)
     label.fill(1)
     count = 0
+    timer = Timer()
     for i in range(niter):
         x = xp.asarray(data)
         y = xp.asarray(label)
         
         if args.arch == 'googlenet':
-            start = xp.cuda.Event()
-            end = xp.cuda.Event()
-            start.record()
+            timer.preprocess()
```

Comment:
And I have also tested this script on GPU, time seems the same as original. Thanks